tests/end-to-end/simple/App.cs

```diff
@@ -0,0 +1,203 @@
+            {
+                {
+                    "True", new Dictionary<string, bool>
+                    {
+                        { "true", true },
+                    }
+                },
+                {
+                    "False", new Dictionary<string, bool>
+                    {
+                        { "false", false },
+                    }
+                },
+            };
+            var lists = new Dictionary<string, Dictionary<string,string[]>> 
+            {
+                {
+                    "Candidates", new Dictionary<string, string[]>
+                    {
+                        {
+                            "Empty", new string[] 
+                            {
+                            }
+                        },
+
+                        {
+                            "Singleton", new string[] 
+                            {
+                                "One",
+                            }
+                        },
+
+                        {
+                            "Pair", new string[] 
+                            {
+                                "One",
+                                "Two",
+                            }
+                        },
+
+                    }
+                },
+            };
+            var numbers = new Dictionary<string, Dictionary<string,int>> 
+            {
+                {
+                    "Prime", new Dictionary<string, int>
+                    {
+                        { "Eleven", 11 },
+                        { "Thirteen", 13 },
+                        { "Seventeen", 17 },
+                    }
+                },
+            };
+            var strings = new Dictionary<string, Dictionary<string,string>> 
+            {
+                {
+                    "Foos", new Dictionary<string, string>
+                    {
+                        { "Foo1", "Foo1" },
+                        { "Foo2", "Foo2" },
+                    }
+                },
+                {
+                    "Bars", new Dictionary<string, string>
+                    {
+                        { "Bar", "Bar" },
+                    }
+                },
+            };
+            var table = new Dictionary<string, Dictionary<string,object>> 
+            {
+                {
+                    "Values", new Dictionary<string, object>
+                    {
+                        { "Boolean", true },
+                        { "Float", 3.14 },
+                        {
+                            "List", new string[] 
+                            {
+                                "1",
+                                "2",
+                                "3",
+                            }
+                        },
+
+                        { "Number", 42 },
+                        { "String", "Baz" },
+                    }
+                },
+            };
```

Comment:
Somewhat nitty but is there 
a) no way to shrink the whitespace in the mappings? It's nearly double the size of other synthesizers
b) Should we have a newline gap here and a comment to indicate the next 3 booleans are the conditions

Reply:
The first question here should be if this spacing is the C# convention. If so, it's the spacing we should use.

Reply:
@HBobertz , I addressed b).

For the spacing, good news! I just learned there is a way to do index-based initialization in C#, so that already makes it shorter. It's still long though so let me lay out the options. in C#, there are two ways to format your code block in curly braces:
all in one line
```csharp
new Thing { Param1 = 1, Param2 = 2 }
```

OR taking up at least 4 lines (or it would be 3 if there is no text before the opening `{`
```csharp
new Thing
{
    Param1 = 1,
    Param2 = 2,
}
```

So for the mappings, that leaves us with a couple options:

```
            var booleans = new Dictionary<string, Dictionary<string,bool>> 
            {
                ["True"] = new Dictionary<string, bool>
                {
                    ["true"] = true,
                }
                ,
                ["False"] = new Dictionary<string, bool>
                {
                    ["false"] = false,
                },
            };
```
```
            var booleans = new Dictionary<string, Dictionary<string,bool>> 
            {
                ["True"] = new Dictionary<string, bool> { ["true"] = true },
                ["False"] = new Dictionary<string, bool> { ["false"] = false },
            };
``` 
I don't like the second option because there may be several items in the inner-most Dictionary, and that line would get really long. Or we could do something like use one line if there are 1 or 0 items, and break it out into multiple lines if there are more. 

What do you think?

Reply:
I'd vote for option 2

Reply:
option 2 seems fine to me